docker/tedd_low_voltage/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  mosquitto:
+    image: eclipse-mosquitto:2.0
+    restart: always
+    ports:
+      - "1883:1883"
+    expose:
+      - "1883"
+    volumes:
+      - mosquitto_data:/mosquitto/data
+      - mosquitto_log:/mosquitto/log
+      - ./mosquitto/config:/mosquitto/config
+    
+  lv_hmp4040:
```

Comment:
shouldn't be in test stack. we only add supporting services there, see web server

Reply:
done